merlin/study/batch.py

```diff
@@ -134,6 +134,14 @@ def get_node_count(default=1):
         the environment cannot be determined.
     :param returns: (int) The number of nodes to use.
     """
+
+    # If flux is the scheduler, we can get the size of the allocation with this
+    try:
+        get_size_proc = subprocess.run("flux getattr size", shell=True, capture_output=True, text=True)
```

Comment:
@koning  do we need to put a flux version check on this call to find the number of nodes?

Reply:
I think that was standard from the start, but I'll check.

Reply:
Looks like that is relatively new actually, So a version check is required.